templates/web/pages/export_products/export_products.tt.js

```diff
@@ -0,0 +1,72 @@
+var poll_n1 = 0;
+var timeout1 = 5000;
+var job_info_state1;
+
+var poll_n2 = 0;
+var timeout2 = 5000;
+var job_info_state2;
+
+var poll_n3 = 0;
+var timeout3 = 5000;
+var job_info_state3;
+
+var minion_status = {
+	"inactive" : "$Lang{minion_status_inactive}{$lc}",
+	"active" : "$Lang{minion_status_active}{$lc}",
+	"finished" : "$Lang{minion_status_finished}{$lc}",
+	"failed" : "$Lang{minion_status_failed}{$lc}"
+};
+
+(function poll1() {
+    \$.ajax({
+      url: '/cgi/minion_job_status.pl?job_id=$local_export_job_id',
```

Comment:
Same here, $local_export_job_id won't be accessible through the template, unless it has been added to the data $template_data_ref that was passed to the template.